templates/postgresql.service.j2

```diff
@@ -5,9 +5,9 @@ After=network.target
 
 [Service]
 Type=forking
-ExecStart={{ postgres_pg_ctl }} -s -D {{ postgres_data_directory }} start -w -t 120
```

Comment:
Please get rid of the old variable everywhere, e.g. in README.md and tasks/install.yml

Reply:
24954bf7ee35da3c7060b34f58cc92c574afd326 should do that.